packages/component-library-react/src/Accordion/index.tsx

```diff
@@ -0,0 +1,44 @@
+import clsx from 'clsx';
+import { useId, useState } from 'react';
+import React from 'react';
+import { Button, Heading3 } from '..';
+
+export interface AccordionProps {
+  label: string;
+  body: any;
+  locale?: string;
+  expandedAccordion?: boolean;
+}
+
+export const Accordion: React.FC<AccordionProps> = ({ label, body, locale, expandedAccordion = false }) => {
+  const [expanded, setExpanded] = useState(expandedAccordion);
+  const id = useId();
+  return (
+    <div className={clsx('utrecht-accordion')}>
+      <Heading3 className={clsx('utrecht-accordion__label')}>
+        <Button
+          dir={locale}
+          className={clsx('utrecht-accordion__button')}
+          appearance="subtle-button"
+          aria-expanded={expanded}
+          aria-controls={`utrecht-accordion-label-${id}`}
+          id={`${id}-id`}
+          onClick={() => setExpanded(!expanded)}
+        >
+          {label}
+        </Button>
+      </Heading3>
+      {expanded && (
```

Comment:
When you only render the expanded sections, search engines will not be able to index collapsed accordion sections.